Dockerfile

```
@@ -1,32 +1,39 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS base
+FROM registry.access.redhat.com/ubi9/python-312:latest as base
 # Only set variables or install packages that need to end up in the
 # final container here.
+USER root
+
 ENV PATH=/app/bin/:$PATH \
```

Hadolint (2.14.0)

[error] Do not refer to an environment variable within the same `ENV` statement where it is defined.

(DL3044)